### PR TITLE
Addressed customer feedback about ControlBox property

### PR DESCRIPTION
**Pull request by rpetrusha**
# Addressed customer feedback about ControlBox property

This PR addresses the following customer feedback:

The documentation for the [MaximizeBox property](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.form.maximizebox) (and also for MinimizeBox) appears to be incorrect. In order to show these boxes as either coloured or greyed out, the ControlBox property also has to be set to True (I found this by experiment). I don't know if this is a bug in the code or the documentation, but one of them needs fixing.

## Suggested Reviewers

@mairaw, @Tanya-Solyanik 

//cc @dend

### CODE REVIEW

File: xml/System.Windows.Forms/Form.xml [hunk 1]
```diff
@@ -1149,9 +1149,9 @@
...
   
- If you set <xref:System.Windows.Forms.Form.ControlBox%2A> to `false`, and also set the <xref:System.Windows.Forms.Form.Location%2A> property, the <xref:System.Windows.Forms.Form.Size%2A> property of Form will not update to reflect that the non-client area of the form has been hidden. To fix this problem, put the code which alters the <xref:System.Windows.Forms.Form.Location%2A> property to the <xref:System.Windows.Forms.Control.HandleCreated> event.  
+ If you set <xref:System.Windows.Forms.Form.ControlBox%2A> to `false` and also set the <xref:System.Windows.Forms.Form.Location%2A> property, the <xref:System.Windows.Forms.Form.Size%2A> property of Form will not update to reflect that the non-client area of the form has been hidden. To fix this problem, put the code which alters the <xref:System.Windows.Forms.Form.Location%2A> property to the <xref:System.Windows.Forms.Control.HandleCreated> event.
```
Comment (mairaw) [inline xml/System.Windows.Forms/Form.xml]: style Form or lowercase?

will not update -> is not updated?

which alters -> that alters

put the code to the event reads a bit funny. put the code in the event perhaps?

File: xml/System.Windows.Forms/Form.xml [hunk 2]
```diff
@@ -1807,8 +1807,11 @@
...
-  
+ When this property is set to `true`, a small button with a question mark appears in the caption bar to the left of the **Close** button. You can use this button to display help for your application. You can create an event handler for the <xref:System.Windows.Forms.Control.HelpRequested> event to display Help information to the user when the Help button of the form is clicked.  
+
+To display a Help button, the value of the form's <xref:System.Windows.Forms.Form.ControlBox> property must be `true`, its default value.  The <xref:System.Windows.Forms.Form.ControlBox> property determines whether the upper left corner of the caption bar contains controls such as a maximize button, a minimize button, a help button, and a close button. In addition, you must also set the form's <xref:System.Windows.Forms.Form.FormBorderStyle%2A> property to <xref:System.Windows.Forms.FormBorderStyle.FixedSingle?displayProperty=nameWithType>, <xref:System.Windows.Forms.FormBorderStyle.Sizable?displayProperty=nameWithType>, <xref:System.Windows.Forms.FormBorderStyle.Fixed3D?displayProperty=nameWithType>, or <xref:System.Windows.Forms.FormBorderStyle.FixedDialog?displayProperty=nameWithType>.
```
Comment (mairaw) [inline xml/System.Windows.Forms/Form.xml]: remove extra space between sentences

upper left corner -> upper-left corner?

**Review by mairaw — APPROVED** [review]
Content-wise changes look good except for a couple of nits I left for you to consider @rpetrusha.

File: xml/System.Windows.Forms/Form.xml (hunk 1 above)
Comment (rpetrusha) [reply xml/System.Windows.Forms/Form.xml]: Other than removing a stray comma, I hadn't even read this horrible sentence. Thanks, @mairaw. For your last comment, "...in the event handler..."

Comment (Tanya-Solyanik) [reply xml/System.Windows.Forms/Form.xml]: Suggestion:  put the code in the event handler.

**Comment by rpetrusha** [comment]
Thanks for the careful review, @mairaw. I've incorporated all of your suggestions.

**Comment by mairaw** [comment]
Looks great now! Feel free to merge when finishes building.

**Pull request merged**